### PR TITLE
Correcting mdlint errors in CI ReadMe

### PR DESCRIPTION
**Pull request by RandyLevensalor**
#### What does this PR do?
Formatting changes to CI ReadMe
Increases timeout for CI verification tests
Adds some additional files to .gitignore

#### Do you have any concerns with this PR?
No

#### How can the reviewer verify this PR?
Read the rendered version of the file

#### Questions:
- Have you connected this PR to the issue it resolves?
No
- Does the documentation need an update?
Yes
- Does this add new Python dependencies?
No
- Have you added unit or functional tests for this PR?
No
- Does this patch update any configuration files?
No

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -41,70 +42,81 @@ document.
...
+    admin_user: admin
+    admin_proj: admin
+    admin_pass: something
+    auth_url: http://10.197.113.30:5000
+    id_api_version: 3
+    proxy_host:
+    proxy_port:
+    ssh_proxy_cmd:
+
+    ext_net: external
```
Comment (spisarski) [inline ci/README.md]: may want to change this to 'public1'

File: ci/README.md [hunk 2]
```diff
@@ -41,70 +42,81 @@ document.
...
+    ext_subnet: external-net
```
Comment (spisarski) [inline ci/README.md]: may want to change this to 'public1-subnet'

File: ci/README.md [hunk 3]
```diff
@@ -41,70 +42,81 @@ document.
...
+
+    src_snaps_boot_dir: /home/ubuntu/snaps-boot
```
Comment (spisarski) [inline ci/README.md]: This value is no longer being used and can be removed

File: ci/README.md [hunk 4]
```diff
@@ -41,70 +42,81 @@ document.
...
+    src_copy_dir: /tmp
+    local_snaps_boot_dir: /home/ubuntu/snaps-boot
```
Comment (spisarski) [inline ci/README.md]: may want to change this to something like { local snaps-boot dir }

File: ci/README.md [hunk 5]
```diff
@@ -41,70 +42,81 @@ document.
...
+
+    os_pxe_user_pass: password
+
+    ctrl_ip_prfx: 10.0.0
+    admin_ip_prfx: 10.1.0
+    priv_ip_prfx: 10.1.1
+    pub_ip_prfx: 10.1.2
+
+    build_kp_pub_path: /tmp/pxe-kp-lab1-launcher.pub
```
Comment (spisarski) [inline ci/README.md]: both filenames should probably read 'lab3'.
We may want to consider leveraging the build_num value here as concurrent jobs using this file ultimately will interfere with each other.

**Comment by RandyLevensalor** [comment]
Run Tests

**Pull request merged**